dashboard/utils.py

```diff
@@ -1,15 +1,15 @@
 import credstash
 import os
 
-def get_secret(secret, context):
-    """Fetch secret from credstash, environment, or return None."""
-    try:
-        res = credstash.getSecret(
-            name=secret,
+
+def get_secret(secret_name, context):
+    """Fetch secret from environment or credstash."""
+    secret = os.getenv(secret_name, None)
+
+    if secret is None:
```

Comment:
nit: Assuming that you are interested here for the value of `secret`, I would go for `if not secret`. This way it's going to work in future changes if you ever change the initialization to something different than None eg an empty string.

Reply:
+1